README.md

```diff
@@ -208,51 +236,80 @@ local:
 
 `{MAJOR_RELEASE}.{MINOR_RELEASE}.{PATCH_RELEASE}`
 
-with all releases up until 1.0.0 considered beta.
-
-`astro-cli` tightly tracks the platform release versioning, this means that compatibility is only guaranteed between matching **minor** versions of the platform and the astro-cli. ie. astro-cli `v0.9.0` is guaranteed to be compatible with houston-api `v0.9.x` but not houston-api `v0.10.x`
+All Astro CLI releases prior to 1.0.0 are considered beta.
 
 ## Debug
 
-`astro-cli` has a debug flag that allows to see queries and different helpful logs that are done internally. You can enable this by passing `--verbosity=debug` to your commands or you can edit your `~/.astro/config.yaml` and add the following to it. This will turn on debug for all requests that are done until it is changed to `info` or removed from config.
+The Astro CLI includes a debug flag that allows you to view queries and internal logs. To enable it, you can pass `--verbosity=debug` in your commands, or you can add the following entry to your `~/.astro/config.yaml` file:
 
 ```yaml
 verbosity: debug
 ```
+Adding this entry to your `~/.astro/config.yaml` file turns on debugging for all requests until you change it to `info`, or you remove it from the file.
+
+## Request a documentation change
+
+If you notice something in our documentation that is wrong, misleading, or could use additional context, the easiest way to make an impact is to create a GitHub issue in this repository.
+
+GitHub issues are triaged by the Astronomer documentation team and addressed promptly. After you've created a GitHub issue, our team may follow up with you with additional questions or comments. Once our team has addressed it, you'll get a notification from GitHub that the issue has been closed and that a change is now live.
+
+1. Go to [Issues](https://github.com/astronomer/docs/issues).
+
+2. Select **New Issue**.
+
+3. Depending on the change you want implemented, select a GitHub issue template.
+
+4. Tell us what you think we can do better by answering the questions in the template.
+
+## Contribute
+
+If you'd like to contribute to Astronomer documentation, you're welcome to create a Pull Request (PR) to this repository with your suggested changes.
+
+1. Fork the repository.
+
+2. Create a branch from `main`.
+
+3. Make your changes in your branch.
+
+4. Submit a PR for review.
+
+    After you've submitted a PR for your changes, Netlify will add a comment to your PR that includes a link to a staging website with your changes.
+
+    Small edits and typo fixes don't need to be linked to an issue and should be merged quickly. To get a timely review on a larger contribution, we recommend first creating a detailed GitHub issue describing the problem and linking that within your PR.
+
+    Every update to the `main` branch of this repository triggers a rebuild of our production documentation page at https://www.docs.astronomer.io. It might take a few moments for your merged changes to appear.
```

Comment:
This isn't relevant to the Astro CLI project, it should rather be part of the `astronomer/docs` repo, at best if you want you can add these sections to the `astronomer/docs` repo and leave a link to that over here.

Reply:
@neel-astro @paolaperaza requested that we add this information to the README. You can review her comment in the originating issue. This information is a duplicate of what already appears in the astronomer/docs repo. I'll remove it.

Reply:
ideally, Astro CLI contribute section should have information on how to contribute to the Astro CLI repo only, not sure if @paolaperaza also meant the same thing in the original issue description